go.mod

```diff
@@ -4,10 +4,9 @@ require (
 	github.com/Shopify/sarama v1.21.0
 	github.com/aws/aws-sdk-go v1.21.8
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
+	github.com/elastic/go-elasticsearch/v6 v6.8.3-0.20190731061920-efbed2e4c2f8
```

Comment:
You should go with the latest version of the library. It should be compatible with older versions that are still supported.